ddtrace/tracer/option.go

```diff
@@ -220,6 +224,17 @@ func newConfig(opts ...StartOption) *config {
 	c.profilerEndpoints = internal.BoolEnv(traceprof.EndpointEnvVar, true)
 	c.profilerHotspots = internal.BoolEnv(traceprof.CodeHotspotsEnvVar, true)
 
+	schemaVersionStr := os.Getenv(namingschema.SpanAttributeSchemaEnvVar)
+	if v, ok := namingschema.ParseVersion(schemaVersionStr); ok {
+		namingschema.SetVersion(v)
+		c.spanAttributeSchemaVersion = int(v)
+	} else {
+		namingschema.SetDefaultVersion()
+		v := namingschema.GetVersion()
```

Comment:
We can make `SetDefaultVersion()` return the value.
```suggestion
		v := namingschema.SetDefaultVersion()
```